docs/management_api.md

```diff
@@ -59,7 +59,7 @@ curl -v -X POST "http://localhost:8081/models?initial_workers=1&synchronous=fals
 The synchronous call returns with HTTP code 200 after all workers have been adjusted.
 
 ```bash
-curl -v -X POST "http://localhost:8081/models?initial_workers=1&synchronous=true&url=https://<s3_path>/squeezenet_v1.1.mar"
+curl -v -X POST "http://localhost:8081/models?initial_workers=1&synchronous=true&url=https://<cloud_OR_noncloud_storage>/squeezenet_v1.1.mar"
```

Comment:
Replace cloud_OR_noncloud_storage with links for squeezenet here https://github.com/pytorch/serve/blob/master/docs/model_zoo.md